python/Ganga/test/GPI/Parser/TestSelect.py

```diff
@@ -50,7 +50,7 @@ def test_c_SelectSJTests(self):
         """ Is is a bird is it a plane... no it's a test for selecting subjobs now"""
         from Ganga.GPI import jobs, Job, ArgSplitter
         
-        j=Job(splitter=ArgSplitter(args=job_names))
+        j=Job(splitter=ArgSplitter(args=[[_] for _ in job_names]))
```

Comment:
The args are expected to be a list of lists not a list of strings from what I can tell. I think a list of strings would meet most of the requirements but I hit a problem at least once that a string (whilst looking like a list) isn't a list.